cypress/integration/birth.spec.ts

```diff
@@ -16,87 +16,189 @@ context('Birth Integration Test', () => {
     indexedDB.deleteDatabase('OpenCRVS')
   })
 
-  it('Tests from application to registration using minimum input', () => {
-    cy.initializeFakeTimers()
-    cy.registerApplicationWithMinimumInput('Tahmid', 'Rahman')
+  
+  // MINImum
+  it('Tests from application to certification using minimum input', () => {
+    //cy.initializeFakeTimers()
+    cy.declareApplicationWithMinimumInput('Arifuz', 'Antor')
+    cy.wait(1000)
```

Comment:
Doesn't have to be done now before merging, but eventually, we'll need to remove all `cy.wait()` calls. We used to have a lot of them at one point when our tests started randomly failing because the CI machine wasn't sometimes fast enough to update the app before moving onto the test assertion. Waits also significantly increase the runtime of the whole test suite.

`cy.get` automatically waits until the element is found and the timeout time can be configured. If there's some type of a change in the app you're waiting for, it's better to use some other way of detecting and waiting for the change. Think "how would the user know loading declarations is done?"